data/json/items/gunmod/underbarrel.json

```diff
@@ -20,6 +20,38 @@
     "min_skills": [ [ "weapon", 2 ] ],
     "flags": [ "BIPOD", "SLOW_WIELD" ]
   },
+  {
+    "id": "m3_tripod",
+    "type": "GUNMOD",
+    "name": { "str": "M3 tripod" },
+    "description": "This solid-metal tripod is a weapon mount used on the M2HB Browning machine gun and the Mk19 grenade launcher, allowing them to be fired on any terrain, but requires the user to be either in a prone or crouching position.",
+    "weight": "22800 g",
+    "volume": "8500 ml",
+    "price": 68000,
+    "price_postapoc": 1000,
+    "install_time": "2 m",
+    "longest_side": "123 cm",
+    "to_hit": -3,
+    "bashing": 5,
+    "looks_like": "bipod",
+    "material": [ "steel" ],
+    "symbol": "A",
+    "color": "brown",
+    "location": "underbarrel",
+    "mod_targets": [ "m2browning", "mark19" ],
+    "handling_modifier": 23,
+    "min_skills": [ [ "weapon", 3 ] ],
+    "flags": [ "TRIPOD", "SLOW_WIELD" ]
+  },
+  {
+    "id": "m205_tripod",
```

Comment:
`"type"` is missing in this item definition.

Reply:
I thought "copy-from" already did that. Anyway, just added it.